python/aitemplate/backend/rocm/gemm/gemm_rcr_bias_add_relu.py

```diff
@@ -128,6 +128,13 @@ def gen_function(
         dim_info_dict,
         "bias_add_relu",
         extra_code=EXTRA_CODE.render(),
+        input_addr_calculator=common.INPUT_ADDR_CALCULATOR.render(
```

Comment:
@ipiszy @chenyang78 I'm just highlighting some change, hopefully to make the review a little easier on your side